build-recipe-dsc

```diff
@@ -95,7 +95,7 @@ dsc_build() {
     # this allows the build environment to be manipulated
     # and alternate build commands can be used
     DSC_BUILD_CMD="$(queryconfig --dist "$BUILD_DIST" --archpath "$BUILD_ARCH" --configdir "$CONFIG_DIR" substitute dsc:build_cmd)"
-    test -z "$DSC_BUILD_CMD" && DSC_BUILD_CMD="dpkg-buildpackage -us -uc"
+    test -z "$DSC_BUILD_CMD" && DSC_BUILD_CMD="LC_ALL=C.UTF-8 dpkg-buildpackage -us -uc"
```

Comment:
Is this the right place?

Shouldn't we change [`build-pkg-deb`](https://github.com/openSUSE/obs-build/blob/9f60ac1fa34fca090dd16dda9c21ba16769830a1/build-pkg-deb#L40) instead?

(but I am not sure who calls who to be honest)

Reply:
Probably both need to be changed; also `build-pkg-deb` needs to consistently run `deb_chroot`.